File: Spring/Chapter7/MyBatisSpringJavaInit/src/main/java/pers/oneToMany/dao/UserWithOrdersDao.java
package pers.oneToMany.dao;

import pers.oneToMany.pojo.UserOrdersPOJO;
import pers.oneToMany.entity.UserWithOrders;

import java.util.List;

public interface UserWithOrdersDao {
    UserWithOrders selectUserOrders1(Integer id);
    UserWithOrders selectUserOrders2(Integer id);
    List<UserOrdersPOJO> selectUserOrders3(Integer id);
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/main/kotlin/pers/oneToMany/dao/UserWithOrdersDao.kt
package pers.oneToMany.dao

import pers.oneToMany.entity.UserWithOrders
import pers.oneToMany.pojo.UserOrdersPOJO

interface UserWithOrdersDao {
    fun selectUserOrders1(id:Int):UserWithOrders
    fun selectUserOrders2(id:Int):UserWithOrders
    fun selectUserOrders3(id:Int):List<UserOrdersPOJO>
}